lib/dialects/postgres/index.js

```diff
@@ -127,7 +127,7 @@ class Client_PG extends Client {
   }
 
   _parseVersion(versionString) {
-    return /^PostgreSQL (.*?)( |$)/.exec(versionString)[1];
```

Comment:
Note to self: it's worse than I thought: `versionString` comes back as

`CockroachDB CCL v21.2.0-beta.4-20-g3061813685 (x86_64-unknown-linux-gnu, built 2021/10/15 07:02:45, go1.16.6)`

So the actual result of `pg --version` would still not match but we're not even getting that in this function.

Reading out `resp.rows` where this string comes from, I get 

~~~
[{"version":"CockroachDB CCL v21.2.0-beta.4-20-g3061813685 (x86_64-unknown-linux-gnu, built 2021/10/15 07:02:45, go1.16.6)"}]
~~~